### PR TITLE
Add test for trailing whitespace

### PR DESCRIPTION
**Pull request by yous**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -73,6 +74,26 @@ if(unnecessities.length > 0 || unnecessitiesInKeywords.length > 0) {
...
+var offenses = []
+rawData.split("\n").forEach(function(line) {
+  if(line.match(/\s+$/)) {
+    offenses.push(line);
+  }
+})
+
+if(offenses.length > 0) {
+  offenses.forEach(function(line) {
+    console.log('"' + line + '" has trailing whitespace.')
```
Comment (muan) [inline test.js]: `has "a" trailing whitespace`

**Comment by muan** [comment]
NICE thank you so much :heart:

File: test.js (hunk 1 above)
Comment (yous) [inline test.js]: Changed and rebased! :smiley:

**Comment by muan** [comment]
Hey looks like there are conflicts, can you resolve them please? :zap:

**Comment by muan** [comment]
Oh kewl

**Comment by muan** [comment]
Hai I made it so that it shows line number instead of the text as I thought it looked quite messy with all the quotes.

image

Thank you! :)

**Comment by yous** [comment]
Nice! :sparkles:
